### PR TITLE
Update HubSpot.xml

### PR DESCRIPTION
**Pull request by koops76**
#10782

### CODE REVIEW

File: src/chrome/content/rules/HubSpot.xml [hunk 1]
```diff
@@ -150,9 +145,6 @@ Non-2xx HTTP code: http://forums.hubspot.com/ (200) => https://hubspot3.vanillaf
 	<rule from="^http://cdn1\.hubspot\.com/"
 		to="https://s3.amazonaws.com/cdn1.hubspot.com/" />
 
-	<rule from="^http://forums\.hubspot\.com/"
-		to="https://hubspot3.vanillaforums.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/HubSpot.xml]: It redirects now to `https://community.hubspot.com/`.

File: src/chrome/content/rules/HubSpot.xml [hunk 2]
```diff
@@ -45,6 +41,7 @@ Non-2xx HTTP code: http://forums.hubspot.com/ (200) => https://hubspot3.vanillaf
 
 	² Refused
 	* 503, akamai
+	⁴ 404
```
Comment (J0WI) [inline src/chrome/content/rules/HubSpot.xml]: `hubspot3.vanillaforums.com` is a 404, but `forums.hubspot.com` serves a mismatching cert.

File: src/chrome/content/rules/HubSpot.xml [hunk 3]
```diff
@@ -54,7 +51,7 @@ Non-2xx HTTP code: http://forums.hubspot.com/ (200) => https://hubspot3.vanillaf
 			- br ²
 			- cdn1			(akamai 503, valid cert)
 			- dev			(works, akamai)
-			- forums ²
+			 ²
```
Comment (J0WI) [inline src/chrome/content/rules/HubSpot.xml]: Please also remove the `²`

**Comment by koops76** [comment]
@J0WI Everything is fixed.

**Pull request merged**